### PR TITLE
Add qemu-upstream branch

### PR DESCRIPTION
**Pull request by gaborigloi**
Signed-off-by: Gabor Igloi <gabor.igloi@citrix.com>

### CODE REVIEW

File: files/init-container.sh [hunk 1]
```diff
@@ -20,6 +20,9 @@ case $GIT_BRANCH in
 	feature/CBT)
 		XS_REPO_KEY=aHR0cDovL3hhcGktcmVwb3MuczMtd2Vic2l0ZS11cy1lYXN0LTEuYW1hem9uYXdzLmNvbS9mZWE3NjJlNy1jYjcwLTRiZTktZWY4Ni00M2FlODlmOTFjZDIvZG9tYWluMA==
 		;;
+	qemu-upstrem)
```
Comment (robhoes) [inline files/init-container.sh]: spelling 😮

Comment (gaborigloi) [reply files/init-container.sh]: Oops well-spotted, thanks :smile:

**Comment by gaborigloi** [comment]
The corresponding PR to upload-scripts: https://github.com/xenserver/upload-scripts/pull/17

**Review by Frezzle — APPROVED** [review]
Looks good; merging.

**Comment by gaborigloi** [comment]
The build is green now: https://travis-ci.org/xapi-project/xenopsd/builds/272481700 :)

**Pull request merged**